### ISSUE TITLE
VF instances do not match static ttfs

### ISSUE THREAD

**Issue by m4rc1e** [googlefonts/mavenproFont#3]
I need to regen the VF so it has an avar table 

![screen shot 2018-06-07 at 10 11 58](https://user-images.githubusercontent.com/7525512/41090084-4f2ebc36-6a3b-11e8-8932-ddbec7e459b0.png)

**Comment by m4rc1e** [comment]
avar has been added.

![screen shot 2018-06-07 at 10 27 37](https://user-images.githubusercontent.com/7525512/41090980-6c5815f8-6a3d-11e8-95c4-24f649fbfa12.png)

Unfortunately, it looks like the kerning isn't working for the VF. 

![mavenpro](https://user-images.githubusercontent.com/7525512/41091671-02ceddf4-6a3f-11e8-9748-db0297beffe9.gif)

The font does have gpos kerning.

![screen shot 2018-06-07 at 10 42 15](https://user-images.githubusercontent.com/7525512/41091888-89f833d4-6a3f-11e8-99fc-c45aad1c9aff.png)

but it's not working.

I've checked the kerning for the other VFs and there isn't an issue.

cc @anthrotype
